common/config.go

```diff
@@ -376,12 +377,47 @@ type JsonRpcUpstreamConfig struct {
 }
 
 type EvmUpstreamConfig struct {
-	ChainId                  int64       `yaml:"chainId" json:"chainId"`
-	NodeType                 EvmNodeType `yaml:"nodeType,omitempty" json:"nodeType"`
-	StatePollerInterval      Duration    `yaml:"statePollerInterval,omitempty" json:"statePollerInterval" tstype:"Duration"`
-	StatePollerDebounce      Duration    `yaml:"statePollerDebounce,omitempty" json:"statePollerDebounce" tstype:"Duration"`
-	MaxAvailableRecentBlocks int64       `yaml:"maxAvailableRecentBlocks,omitempty" json:"maxAvailableRecentBlocks"`
-	GetLogsMaxBlockRange     int64       `yaml:"getLogsMaxBlockRange,omitempty" json:"getLogsMaxBlockRange"`
+	ChainId                            int64       `yaml:"chainId" json:"chainId"`
+	NodeType                           EvmNodeType `yaml:"nodeType,omitempty" json:"nodeType"`
+	StatePollerInterval                Duration    `yaml:"statePollerInterval,omitempty" json:"statePollerInterval" tstype:"Duration"`
+	StatePollerDebounce                Duration    `yaml:"statePollerDebounce,omitempty" json:"statePollerDebounce" tstype:"Duration"`
+	MaxAvailableRecentBlocks           int64       `yaml:"maxAvailableRecentBlocks,omitempty" json:"maxAvailableRecentBlocks"`
+	GetLogsAutoSplittingRangeThreshold int64       `yaml:"getLogsAutoSplittingRangeThreshold,omitempty" json:"getLogsAutoSplittingRangeThreshold"`
+	GetLogsMaxAllowedRange             int64       `yaml:"getLogsMaxAllowedRange,omitempty" json:"getLogsMaxAllowedRange"`
+
+	// TODO: remove deprecated alias (backward compat): maps to GetLogsAutoSplittingRangeThreshold
+	GetLogsMaxBlockRange int64 `yaml:"getLogsMaxBlockRange,omitempty" json:"getLogsMaxBlockRange"`
+}
+
+func (e *EvmUpstreamConfig) UnmarshalYAML(unmarshal func(interface{}) error) error {
+	type rawEvmUpstreamConfig struct {
+		ChainId                            int64       `yaml:"chainId" json:"chainId"`
+		NodeType                           EvmNodeType `yaml:"nodeType,omitempty" json:"nodeType"`
+		StatePollerInterval                Duration    `yaml:"statePollerInterval,omitempty" json:"statePollerInterval"`
+		StatePollerDebounce                Duration    `yaml:"statePollerDebounce,omitempty" json:"statePollerDebounce"`
+		MaxAvailableRecentBlocks           int64       `yaml:"maxAvailableRecentBlocks,omitempty" json:"maxAvailableRecentBlocks"`
+		GetLogsAutoSplittingRangeThreshold int64       `yaml:"getLogsAutoSplittingRangeThreshold,omitempty" json:"getLogsAutoSplittingRangeThreshold"`
+		GetLogsMaxAllowedRange             int64       `yaml:"getLogsMaxAllowedRange,omitempty" json:"getLogsMaxAllowedRange"`
+
+		// TODO: remove deprecated alias (backward compat): maps to GetLogsAutoSplittingRangeThreshold
+		GetLogsMaxBlockRange int64 `yaml:"getLogsMaxBlockRange,omitempty" json:"getLogsMaxBlockRange"`
```

Comment:
```suggestion
		GetLogsMaxBlockRange int64 `yaml:"getLogsMaxBlockRange,omitempty" json:"-"`
```

So that it won't appear in erpc_project method where we print the current config

Reply:
It would be good if we can eliminate it easily in the Typescript types as well, if it's too complicated it's ok we can live with it for now.